documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -143,21 +143,21 @@ You configure Hibernate OGM and Infinispan in two steps basically:
 * If you need to run JPQL or HQL queries, add Hibernate Search on the classpath
   (<<ogm-query-using-hibernate-search>>)
 
-Note that, except when using [acronym]`JNDI`, Hibernate OGM will bootstrap and Infinispan Embedded node
-in your same JVM, and terminate it on shutdown of the Hibernate instance.
+Note that, except when using [acronym]`JNDI`, Hibernate OGM will bootstrap Infinispan Embedded 
+in the same JVM and terminate it on shutdown of the Hibernate OGM factory.
```

Comment:
I think you mean `session factory` here? `factory` is a bit weird as it could be a `factory` of anything.

Reply:
I will replace it with `session factory`